html/index.html

```diff
@@ -1,17 +1,337 @@
+<div class="team-area-section">
+	<div class="team-area">
+		<div class="our-experts">
+			<h3>Our expert doctors</h3>
+		</div>
+		<div class="teams">
+			<div class="team-one">
+				<img src="images/1.png" alt="">
+				<h4>Dr. Khaled Hossen</h4>
+				<h5>Physical Medicine, MBBS</h5>
+				<p>Pharetra ultricies aenean, sit metus integer arcu turpis dndimentum pellentesque world</p>
+				<div class="social-icons-wrap">
+					<a href="https://www.instagram.com/" class="facebook" target="_blank">
+						<i class="fa fa-facebook" aria-hidden="true"></i>
+					</a>
+
+					<a href="twitter" class="twitter">
+						<i class="fa fa-twitter" aria-hidden="true"></i>
+					</a>
+
+					<a href="google-plus" class="google-plus">
+						<i class="fa fa-google-plus-official" aria-hidden="true"></i>
+					</a>
+				</div>
+			</div>
+
+
+			<div class="team-one">
+				<img src="images/2.png" alt="">
+				<h4>Dr. Shipon Kumar</h4>
+				<h5>Physical Medicine, MBBS</h5>
+				<p>Pharetra ultricies aenean, sit metus integer arcu turpis dndimentum pellentesque world</p>
+				<div class="social-icons-wrap">
+					<a href="https://www.instagram.com/" class="facebook" target="_blank">
+						<i class="fa fa-facebook" aria-hidden="true"></i>
+					</a>
+
+					<a href="twitter" class="twitter">
+						<i class="fa fa-twitter" aria-hidden="true"></i>
+					</a>
+
+					<a href="google-plus" class="google-plus">
+						<i class="fa fa-google-plus-official" aria-hidden="true"></i>
+					</a>
+				</div>
+
+			</div>
+			<div class="team-one">
+				<img src="images/3.png" alt="">
+				<h4>Dr. Marina Yigo</h4>
+				<h5>Physical Medicine, MBBS</h5>
+				<p>Pharetra ultricies aenean, sit metus integer arcu turpis dndimentum pellentesque world</p>
+				<div class="social-icons-wrap">
+					<a href="https://www.instagram.com/" class="facebook" target="_blank">
+						<i class="fa fa-facebook" aria-hidden="true"></i>
+					</a>
+
+					<a href="twitter" class="twitter">
+						<i class="fa fa-twitter" aria-hidden="true"></i>
+					</a>
+
+					<a href="google-plus" class="google-plus">
+						<i class="fa fa-google-plus-official" aria-hidden="true"></i>
+					</a>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
+
+<div class="signup-section">
+	<h5>Signup for Our Newslatter</h5>
+
+	<div class="newsletter-area">
+		<form action="#" class="wow fadeInUp" data-wow-delay="0.4s">
+			<input type="email" placeholder="Enter Your Email Address" required="">
+			<button type="submit" class="primary-light-bg"><i class="fa fa-paper-plane"></i></button>
+		</form>
+	</div>
+</div>
+
+<div class="latest-update-section">
+	<div class="wrapper">
+		<div class="latest-update-section-one">
+			<h2>Latest Update</h2>
+			<p>Condimentum rutrum placerat egestas condimentum mi eros. Eleifend cras quirntum</br>Feugiat elit
+				placerat. Diam tempor malesuada</p>
+		</div>
+		<div class="latest-update-section-two">
+			<div class="admin-section">
+				<img src="images/1 (1).jpg" alt="admin">
```

Comment:
image names should be separated using '-'
e.g. 
`<img src="images/1 (1).jpg" alt="admin">`
Should be :
`<img src="images-one.jpg" alt="admin">`